docker/image_types.yaml

```diff
@@ -2,34 +2,44 @@
 # dependency graph.
 
 'base-image':
+  # base: "True"
   tag: 'base-image'
   context: 'docker/base-image'
 
 'base-builder':
+  # base: "True"
   tag: 'base-builder'
   context: 'docker/base-builder'
   depends_on:
     - 'base-image'
 
 'base-runner':
+  # base: "True"
   tag: 'base-runner'
   context: 'docker/base-runner'
   depends_on:
     - 'base-image'
 
-'coverage-builder':
-  tag: 'builders/coverage'
-  context: 'fuzzers/coverage'
-  dockerfile: 'fuzzers/coverage/builder.Dockerfile'
+'dispatcher-image':
+  # base: "True"
+  tag: 'dispatcher-image'
+  context: 'docker/dispatcher-image'
   depends_on:
-    - 'base-builder'
+    - 'base-image'
 
-'coverage-{benchmark}-builder':
-  tag: 'builders/coverage/{benchmark}'
-  context: '.'
-  dockerfile: 'docker/benchmark-builder/Dockerfile'
-  depends_on:
-    - 'coverage-builder'
+# 'coverage-builder':
```

Comment:
Commented out?

Reply:
Yes, need suggestion: Should coverage have a different image type? Or is it fine if the required runners are removed from set of buildable images ?

Reply:
Ideally it would be the same image type and making this the case may be the next task. But for now, the goal is one source of truth, so if you need a different image to get it working, then do that. I'm not sure I understand the second question.

Reply:
It works the way it is now. Second question is basically : I'm (currently) removing the runners for coverage images. Is that fine  or does a separate coverage image type (commented) useful in the long run?

Reply:
>It works the way it is now.

It will work in an experiment? I'm skeptical of this.
It might be easiest to fix the experiment code to handle this though.

> I'm (currently) removing the runners for coverage images. Is that fine or is a separate coverage image type (commented) useful in the long run?

Coverage runners won't ever be used. If we have to build them anyway for simplicity that's fine. But I don't think they need to be a seperate image. I'm confused why you had them at all since we never had coverage runners in make or gcb yaml.

Reply:
> It will work in an experiment? I'm skeptical of this.
> It might be easiest to fix the experiment code to handle this though.

I meant that the coverage image type needn't be separate here (as ideal).

> Coverage runners won't ever be used. If we have to build them anyway for simplicity that's fine. But I don't think they need to be a seperate image. I'm confused why you had them at all since we never had coverage runners in make or gcb yaml.

No, it doesn't have coverage runners at all. I removed the coverage image type from #543 therefore asked for review on that.

Reply:
Not sure why the coverage type was removed.

> I'm (currently) removing the runners for coverage images.

If you keep the coverage type, you wouldn't need to do that, no? Pls see previous discussion at https://github.com/google/fuzzbench/pull/543#discussion_r458354778, and let me know if that makes sense.

Reply:
I agree that adding a coverage image is cleaner. But avoiding a runner is a nested conditional as opposed to 2-3 new image types (for standard + oss-fuzz benchmarks) in the template. So, the current template file treats the coverage as a fuzzer (it is part of `fuzzers/` directory).